airframe-di-macros/src/main/scala-2/wvlet/airframe/SourceCode.scala

```diff
@@ -22,7 +22,8 @@ case class SourceCode(
     private val filePath: String,
     fileName: String,
     line: Int,
-    col: Int
+    col: Int,
+    codeText: String = ""
```

Comment:
Adding a new parameter to the case class will break binary compatibility and caused compilation errors as airframe depends on airframe/airspec. Let's give up showing the code fragment